djangoproject/templates/fundraising/includes/_hero_with_logo.html

```diff
@@ -2,10 +2,16 @@
 <div class="hero">
   <div class="hero-logo">
     {% if obj.url %}<a href="{{ obj.url }}" rel="nofollow">{% endif %}
-    <img src="{% static 'img/fundraising-heart.svg' %}"
-         data-src="{% if obj.logo %}{{ obj.thumbnail.url }}{% else %}{% static 'img/fundraising-heart.svg' %}{% endif %}"
-         data-src-retina="{% if obj.logo %}{{ obj.thumbnail.url|resolution:'2x' }}{% else %}{% static 'img/fundraising-heart.svg' %}{% endif %}"
-         alt="{{ obj.display_name }}" />
+    {% if obj.logo %}
+      <img
+        src="{{ obj.thumbnail.url }}"
```

Comment:
If I understood the `srcset` page on MDN, it should be as easy as:
```suggestion
        src="{{ obj.thumbnail.url }}"
        srcset="{{ obj.thumbnail.url|resolution:'2x' }} 2x"
```

Reply:
Done in [8aa3112](https://github.com/django/djangoproject.com/pull/1784/commits/8aa3112a9aa11fe3673a816c356c0359018631c6)!